stories/Molecules/Text/SmallCopy/SmallCopy.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './smallcopy.scss';
+import { P } from '../../../Atom/Base-typography/Paragraph/p';
+
+export const Smallcopy = ({ locationName, locationText, phoneName, phonenumber, emailName, emailid }) => {
+    return (
+        <div className='small__lockup'>
+            <div className='small__lockup_location'>
+                <span>{locationName}</span>
+                <P label={locationText}/>
+            </div>
+            <div className='small__lockup_caption'>
+                <span>{phoneName}:</span> <P label={phonenumber}/>
+            </div>
+            <div className='small__lockup_caption'>
+                <span>{emailName}:</span> <P label={emailid}/>
+            </div>
+        </div>
+    );
+};
```

Comment:
@Shubham-Sharma-77 seems like you have not checked the code by running linting so please check that and also correct the indentation to 2 spaces